docs/source/install.rst

```diff
@@ -1,15 +1,57 @@
 Install Dask
 ============
 
-You can install dask using ``conda``::
+Dask can be installed from source, or can be installed with a single command with 
+pip or with `Anaconda <https://www.continuum.io/downloads>`_.
```

Comment:
Isn't `conda` the relevant command/tool?  I recommend changing the text to `conda` but keeping the download link the same.